mantle/platform/qemu.go

```diff
@@ -277,70 +280,99 @@ func findLabel(label, pid string) (string, error) {
 	return strings.TrimSpace(string(stdout)), nil
 }
 
-// setupIgnition copies the ignition file inside the disk image.
-func setupIgnition(confPath string, diskImagePath string) error {
+type coreosGuestfish struct {
```

Comment:
Also factored out a guestfish API since I originally started to make it a separate function.